src/main/java/com/backend/controller/AssetRefDataController.java

```diff
@@ -38,7 +38,7 @@ public AssetRefDataController(IAssetRefDataService assetRefDataService) {
     //     return response;
     // }
 
-    @GetMapping(path = "/asset/{assetId}")
+    @GetMapping(path = "/assetref/{assetId}")
```

Comment:
suggestion: probably better to call this `"/asset/ref/{asset_id}"`

Reply:
updated! thanks david